enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -0,0 +1,255 @@
+| Request               |log event metadata and request body but not response body. This does not apply for non-resource requests.|
+| RequestResponse       |log event metadata, request and response bodies. This does not apply for non-resource requests.|
+
+
+OpenShift audit profiles provide an abstraction layer to the Kube `policy` API, where each profile corresponds to an entire predefined policy object. These policies are maintained by OpenShift and encapsulate various levels of logging coverage, while also ensuring rules for sensitive resources, such as Secret, Route, and OAuthClient objects, are only ever logged at the metadata level. OpenShift OAuth server events are only ever logged at the metadata level.
+
+| OpenShift Audit Policy Profile | Description                                                                                                                                                                                                                           |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| None                           | No requests are logged; even OAuth access token requests and OAuth authorize token requests are not logged.                                                                                                                           |
+| Default                        | Logs only metadata for read and write requests; does not log request bodies except for OAuth access token requests. This is the default policy.                                                                                       |
+| WriteRequestBodies             | In addition to logging metadata for all requests, logs request bodies for every write request to the API servers (create, update, patch, delete, deletecollection). This profile has more resource overhead than the Default profile. |
+| AllRequestBodies               | In addition to logging metadata for all requests, logs request bodies for every read and write request to the API servers (get, list, create, update, patch). This profile has the most resource overhead.                            |
+
+MicroShift will reuse the existing OpenShift custom resource, `apiserver.config.openshift.io`, defined as part of the [OpenShift API](https://github.com/openshift/api/blob/master/config/v1/types_apiserver.go) as part of the translation process from "profile" to "policy". Logic for translating the OpenShift profiles to the `policy.audit.k8s.io` objects is provided by the [openshift/library-go](https://github.com/openshift/library-go/tree/release-4.14/pkg/operator/apiserver/audit) library and is intended for import into external code bases.
```

Comment:
This will be internal, right? We aren't exposing that API?

Reply:
Right, it's just to make use of openshift lgic